### ISSUE TITLE
Printing available function when on runs  `Plugins › OMERO › OMERO Extensions`

### ISSUE THREAD

**Issue by romainGuiet** [GReD-Clermont/omero_macro-extensions#11]
HI,

When one runs `Plugins › Bio-Formats › Bio-Formats Macro Extensions`,  a message pops up specifying that it is "designed to be used in a macro" and then it prints available functions in the Log ([ corresponding code](https://github.com/ome/bioformats/blob/ef26436c2d079f05c1047816e564ffcc63bb0b04/components/bio-formats-plugins/src/loci/plugins/macro/LociFunctions.java#L560)).

A similar feature would be nice for your [omero_macro-extensions](https://github.com/GReD-Clermont/omero_macro-extensions)
 and can be[included in this part of your code](https://github.com/GReD-Clermont/omero_macro-extensions/blob/6efee9ce42fbb93df10d94fa39c9097d2e694359/src/main/java/fr/igred/ij/plugin/OMEROMacroExtension.java#L979).

Cheers,

R

**Comment by ppouchin** [comment]
I will add that as soon as possible. The code is already a bit long, so I'll see if I can put the helper text in an external file or something.

**Comment by romainGuiet** [comment]
I tested the `omero_macro-extensions-1.2.0-print-functions.jar` from artifacts-PR12.zip and the print is exactly what we were looking for ! 👍 

```
To gain access to more advanced features of OMERO
from within a macro, put the following line at the
beginning of your macro:

run("OMERO Extensions");

This will enable the following macro functions:


### Handles connection (should be used first/last) ###

Ext.connectToOMERO(host, port, username, password)
> Connects to the given OMERO `host` (and `port`) 
> using the provided `username` and `password`.

Ext.disconnect()
> Disconnects from the server.


### Changes the context for other commands ###

Ext.switchGroup(groupId)
> Switches to the given group on OMERO.

Ext.listForUser(username)
> Filters listings to only include data from the given user.

Ext.sudo(username)
> Runs subsequent commands as the specified user, 
> if the logged-in user has sudo rights.

Ext.endSudo()
> Switches back to the logged-in user.

...
```

**Comment by ppouchin** [comment]
Solved in #12, released as 1.2.0.
